WooCommerce/Classes/POS/Models/Cart.swift

```diff
@@ -18,6 +18,13 @@ struct CartItem {
 struct CartCouponItem {
     let id: UUID
     let code: String
+    let summary: String
+
+    init(id: UUID, code: String, summary: String = "") {
```

Comment:
```suggestion
    init(id: UUID, code: String, summary: String) {
```
*Will need to update some `CartViewHelperTests` to make the target compile